### ISSUE TITLE
(BUG)  Cannot run tool using docker backend due to mount issue

### ISSUE THREAD

**Issue by logicminds** [puppetlabs/prm#138]
**Describe the bug**
```
prm exec puppetlabs/ra10ke
8:34PM INF Creating new image. Please wait...
8:34PM INF Code path: /home/user1
8:34PM INF Cache path: /home/user1/.pdk/prm/cache
8:34PM INF Additional Args: []
8:34PM ERR Error executing tool puppetlabs/ra10ke: Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /home/user1/.pdk/prm/cache
Error: Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /home/user1/.pdk/prm/cache


**To Reproduce**
On a virgin prm system, install prm and run a tool.

**Expected behavior**
Error not returned

**Comment by binford2k** [comment]
Which containerization backend are you using? I'm having that problem with Colima but not Docker Desktop.

**Comment by logicminds** [comment]
I was using docker.  This was just an issue of the prm tool not creating the cache directory and expecting it.

**Comment by logicminds** [comment]
The prm tool needs to create the cache directory for the user before running.  Otherwise it will not exist until the user creates it manually.

**Comment by petergmurphy** [comment]
Hi logicminds,

This bug should be fixed with the latest release of PRM, `0.2.0`. Feel free to give it a try!

Peter
